packages/frontend/src/config/redux/videoTabs/actions.ts

```diff
@@ -0,0 +1,19 @@
+export enum Actions {
+    SET_ACTIVE_TAB = "set active tab",
+    SET_DEFAULT_TAB = "set default tab"
+}
+
+
+export const setActiveTab = (activeTab: string) => {
+    return{
+        type: Actions.SET_ACTIVE_TAB,
+        payload: activeTab
```

Comment:
payload nie powinien byc stringiem bo utrudnia to przyszly rozwój. Powinien byc obiektem chodzby {}